### ISSUE TITLE
Build failed

### ISSUE THREAD

**Issue by hzlkarakus** [Azure/azure-webjobs-sdk-extensions#804]
Hello,
I've cloned the project and when I try to build it, I'm getting these errors:

Could not write to output file 'C:\****.Twilio.xml' -- 'Access to the path 'C:\****Twilio.xml' is denied.'	
Metadata file 'C:\****\src\WebJobs.Extensions.MobileApps\bin\Debug\netstandard2.0\Microsoft.Azure.WebJobs.Extensions.MobileApps.dll' could not be found.

How can I solve this issue? Is there a solution where I don't have to give access?

**Comment by ramya894** [comment]
@hzlkarakus thanks for reporting, can you provide the repro steps to reproduce the issue.

**Comment by hzlkarakus** [comment]
@ramya894 of course, I cloned the main branch on my local and I'm using VS 2022. When I "Build solution" I get the following errors. 

Build:
<img width="267" alt="Build" src="https://user-images.githubusercontent.com/116739377/198294002-dcdd8ecc-2869-4530-ad15-2644b96bbee0.png">

Errors:

![image](https://user-images.githubusercontent.com/116739377/198295922-e35f296e-a45a-4cc9-8f46-0cd7e4e96669.png)

**Comment by kshyju** [comment]
@mathewc Do you have any thoughts on why the build tries to write files to C drive?

**Comment by hzlkarakus** [comment]
Hi @kshyju and @mathewc , is there any development regarding this issue? Thank you.

**Comment by jviau** [comment]
Engineering has been refactored to use common artifacts. If this issue still persists, it means you have local env vars overriding artifact output directories and you will need to address that.
